src/ReachSets/ReachSets.jl

```diff
@@ -113,6 +115,8 @@ available_algorithms = Dict{String, Dict{String, Any}}()
 # "explicit" backends
 push!(available_algorithms, "explicit_blocks"=>Dict("func"=>reach_blocks!,
                                                     "is_explicit"=>true))
+push!(available_algorithms, "explicit_blocks_mixed"=>Dict("func"=>reach_blocks!,
```

Comment:
Since the element is the same, why not reuse it? You can actually ignore this map completely because it only contains one element. See the next comment.

Reply:
`why not reuse it?` - I just wanted explicitly divide these 2 `reach_blocks`.

Reply:
You can remove this line now because it is not used anymore.